### ISSUE TITLE
cookiejar import issue?

### ISSUE THREAD

**Issue by xiconet** [srveit/mechanize-js#12]
Hello,

First of all, thank you for your work.
I've tried unsuccessfully to run the ``submit_form.js`` with the following error:
```
ReferenceError: Cookie is not defined
    at Object.<anonymous> (C:\Users\ARC\node_modules\mechanize\bin\submit_form.js:11:14)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:393:7)
    at startup (bootstrap_node.js:150:9)
    at bootstrap_node.js:508:3
```
Unfortunately, I don't know much about the nodejs import system.
My OS is Windows7, node version is v6.10.2 and npm is v3.10.10

Thanks in advance for your help

**Comment by mkormendy** [comment]
Try adding 'var' to the beginning of the line where it attempts to declare the new cookies:

`cookie = new Cookie("sessionid=123;domain=.example.com;path=/");`

change to:

`var cookie = new Cookie("sessionid=123;domain=.example.com;path=/");`

**Comment by srveit** [comment]
This is fixed in 1.0.1
